src/Hal/Board.cpp

```diff
@@ -68,6 +68,7 @@ static const IoPin* ioPinList[] =
     &testPinOut,
     &ledMatrixDataOut,
     &ldrIn
+    ,&dhtIn

```

Comment:
The "," should be at the line above.

Reply:
done, i was preparing it like this for a #IFDEF exclusion later on